dev-python/pandas/pandas-0.23.3.recipe

```diff
@@ -0,0 +1,129 @@
+SUMMARY="A powerful data analysis library for Python"
+DESCRIPTION="Pandas is a Python package providing fast, flexible \
+and expressive data structures designed to make working with \
+\"relational\" or \"labeled\" data both easy and intuitive. \
+It aims to be the fundamental high-level building block for \
+doing practical, real world data analysis in Python."
+HOMEPAGE="https://pandas.pydata.org/"
+COPYRIGHT="2008-2012 AQR Capital Management, LLC
+	Lambda Foundry, Inc.
+	PyData Development Team"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://files.pythonhosted.org/packages/3e/56/82c4d4c049294f87ebd05b65fdcbc9ed68bd23fb0a7e4469caf9a75d199f/pandas-0.23.3.tar.gz"
+CHECKSUM_SHA256="9cd3614b4e31a0889388ff1bd19ae857ad52658b33f776065793c293a29cf612"
+SOURCE_DIR="pandas-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	pandas$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+
+PYTHON_PACKAGES=()
+PYTHON_VERSIONS=()
+PYTHON_LIBSUFFIXES=()
+# We don't have python2 for secondaryArch
+if [ -z "$secondaryArchSuffix" ]; then
+	PYTHON_PACKAGES+=(python)
+	PYTHON_VERSIONS+=(2.7)
+	PYTHON_LIBSUFFIXES+=("")
+	BUILD_REQUIRES+="
+		setuptools_python
+		cython_python
+		python_numpy
+		dateutil_python
+		pytz_python
+		"
+	BUILD_PREREQUIRES+="
+		cmd:python2
+		"
+fi
+# gcc2 does not support the flags passed by python3
+if [ "$effectiveTargetArchitecture" != x86_gcc2 ]; then
+	PYTHON_PACKAGES+=(python3)
+	PYTHON_VERSIONS+=(3.6)
+	PYTHON_LIBSUFFIXES+=(m)
+	BUILD_REQUIRES+="
+		setuptools_python3
+		cython_python3
+		python3_numpy
```

Comment:
Suggestion:
```diff
-		python3_numpy
+		python3_numpy$secondaryArchSuffix
```
Okay, this is not required because python3_numpy does not have support for x86 secondary arch.
But this situation might change someday, so this suggestion is probably welcome.